docs/agent/config.md

```diff
@@ -10,7 +10,7 @@ might mean:
 
 ## Environment variables
 
-All options supported by the Infrastructure Agent in the configuration file can also be set through environment variables;
+All options supported by the Infrastructure and Logs Agents in the configuration file can also be set through environment variables:
```

Comment:
I don't believe we should ever mention `Infrastructure` or `Logs` agent. There is one single agent which is the Datadog Agent.

Reply:
I agree, at the same time we need to find a way to document how things are currently working. One way I think I could improve this would be to mention instead that `all options supported in the configuration file can also be set through environment variables, except the options under apm_config and process_config`. What do you think?

note: we do mention "logs agent" in both the datadog.yaml.example file and the output of the agent status command

Reply:
The new sentence is perfect.

I do agree that in the datadog.yaml file and in the status of the agent we show different agent but that can make sens.
You are looking at the technical details and configuration parameters of the Datadog Agent and can configure sub section.

So I think it is fine to have this kind of details when you are diving within the Datadog Agent. But in the documentation we should always refer about Log Collection in the Datadog Agent, or Process Collection, Trace Collection, .... and not individual agent that might leads to confusion as I saw cases where user thought they had to install 4 different agents to collect metrics, traces, logs, and processes.

Reply:
ok, thanks for the explanation, makes sense to me 👍

Reply:
should be addressed, let me know if you have other comments @NBParis